### ISSUE TITLE
Documentation Translations

### ISSUE THREAD

**Issue by fpliger** [pyscript/docs#147]
tldr; documentation seems stable enough for us to consider supporting translations and a process to automate it and have a light review process

In addition to the `tldr`, there are very large communities interested in PyScript and language is actually a barrier. They'd receive great value from a local translation of our documentation. The risks we identified in the past, when having this conversation before, have mainly been solved and the project is mature enough for us to bring this topic back. In addition to that, LLMs are actually not bad at translating text/code with good contextual awareness. 

With all of that in mind, let's do it!

/cc @ntoll
